genome_kit/data_manager.py

```diff
@@ -255,8 +255,8 @@ def upload_file(self, filepath: str, filename: str, metadata: Dict[str, str]=Non
 
     @lru_cache
     def list_available_genomes(self):
-        blobs = self.bucket.list_blobs(match_glob="*.{2bit,cfg}")
-        return [blob.name.rpartition(".")[0] for blob in blobs]
+        blobs = self.bucket.list_blobs()
+        return [blob.name.rpartition(".")[0] for blob in blobs if blob.name.endswith(".2bit") or blob.name.endswith(".cfg")]
```

Comment:
Slight performance hit but the number of files isn't that large, and this isn't the default DataManager.